stdlib/Distributed/test/runtests.jl

```diff
@@ -3,9 +3,11 @@
 # Run the distributed test outside of the main driver since it needs its own
 # set of dedicated workers.
 include(joinpath(Sys.BINDIR, "..", "share", "julia", "test", "testenv.jl"))
+
 disttestfile = joinpath(@__DIR__, "distributed_exec.jl")
+managerstestfile = joinpath(@__DIR__, "managers.jl")
 
-cmd = `$test_exename $test_exeflags $disttestfile`
+cmd = `$test_exename $test_exeflags $disttestfile $managerstestfile`
```

Comment:
I think we can just include this file on its own (e.g. `include("managers.jl")`) on line 15. Does not have to run in a separate process etc.

Reply:
addressed in 4f1f3c9